app/mailers/notification_mailers/also_commented.rb

```diff
@@ -8,6 +8,7 @@ def set_headers(comment_id)
 
       if mail?
         @headers[:from] = "\"#{@comment.author_name} (diaspora*)\" <#{AppConfig.mail.sender_address}>"
+        @headers[:in_reply_to] = @headers[:references] = "<#{@comment.parent.guid}@#{AppConfig.bare_pod_uri}>"
```

Comment:
If the pod doesn't run on the default port (80/443) `AppConfig.bare_pod_uri` returns the domain with `:port`. But if I understand the [specs](https://tools.ietf.org/html/rfc2822#section-3.2.4) correctly, `:` is not a valid character. My suggestion is to only use the domain without the port.